agendas/2020-05-07.md

```diff
@@ -57,4 +58,5 @@ Example agenda item:
 1. Review agenda (2m, Lee)
 1. Review previous meeting's action items (5m, Lee)
    - [Currently open action items](https://github.com/graphql/graphql-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Action+item+%3Aclapper%3A%22)
+   - [#710 - Add namedType and punctuatedName to __Type](https://github.com/graphql/graphql-spec/pull/710)
```

Comment:
Please add an item like this:

```suggestion
1. Discuss RFC adding introspection shortcuts to determine full typename and named type’s name (15m, Terrence)
   - [#710 - Add namedType and punctuatedName to __Type](https://github.com/graphql/graphql-spec/pull/710)
```

Also ensure you have signed the spec membership agreement (see README).

Reply:
Ugh I don’t know why GitHub adds a space when I do this on mobile 🤷‍♂️